utils/rpms/daos.spec

```diff
@@ -65,7 +70,6 @@ BuildRequires: ipmctl-devel
 BuildRequires: python-devel python3-devel
 BuildRequires: Modules
 %if 0%{?is_opensuse}
-# have choice for boost-devel needed by cart-devel: boost-devel boost_1_58_0-devel
 BuildRequires: boost-devel
```

Comment:
Looks like you need this even on EL7 now.  So just move this above line 72 so that it's unconditional now.